scripts/re_enable_gui.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Re-enable GUI
+sudo systemctl set-default graphical.target
```

Comment:
Since this script is just one line, it may be better to just copy this into the documentation directly.  This is a bit more transparent to the user.

Reply:
Ok, modified.

https://github.com/tokk-nv/jetbot/blob/docs/feedbacks/docs/software_setup/docker.md